tests/agent_test.py

```diff
@@ -183,7 +195,12 @@ def testTsunamiAgent_WhenServiceAssetAndTsunamiScanHasVulnerabilities_ShouldRepo
 
     mock_report_vulnerability.assert_called_once_with(entry=kb_entry,
                                                       technical_detail=f'```json\n{data}\n```',
-                                                      risk_rating=agent_report_vulnerability_mixin.RiskRating.HIGH)
+                                                      risk_rating=agent_report_vulnerability_mixin.RiskRating.HIGH,
+                                                      vulnerability_location=agent_report_vulnerability_mixin.VulnerabilityLocation(
+                                                          metadata=[agent_report_vulnerability_mixin.VulnerabilityLocationMetaData(agent_report_vulnerability_mixin.MetaDataType.PORT, '6000')],
+                                                          asset=domain_asset.DomainName(name='https://ostorlab.co')
```

Comment:
If you are using DomainName asset, then i expect the name to be a domain name, not a link.
You can add the link as part of the metadata.